clients/admin-ui/src/features/system/add-multiple-systems/AddMultipleSystems.tsx

```diff
@@ -195,6 +196,17 @@ export const AddMultipleSystems = ({ redirectRoute }: Props) => {
     },
   });
 
+  const {
+    totalRows,
+    onPreviousPageClick,
+    isPreviousPageDisabled,
+    onNextPageClick,
+    isNextPageDisabled,
+    setPageSize,
+    startRange,
+    endRange,
+  } = useClientSidePagination(tableInstance);
```

Comment:
This will function that same as before. It's just updating the table to use the new client side pagination hook.